### PR TITLE
Updating documentation for newly added binary packages

### PR DESCRIPTION
**Pull request by AhmedBM**
## Description

* Added instructions on using pre-built binary packages including enlistment instructions for both `insiders-fast` and `prod` channels.

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,39 @@
...
+Debian 11 | x86_64 | [insiders-fast](./devops/README.md#debian-11---insiders-fast-amd64) / [prod](./devops/README.md#debian-11---prod-amd64)
+Debian 12 | x86_64 | [insiders-fast](./devops/README.md#debian-12---insiders-fast-amd64)
+Red Hat Enterprise Linux 7 | x86_64 | [insiders-fast](./devops/README.md#red-hat-enterprise-linux-7---insiders-fast-amd64)
+Red Hat Enterprise Linux 8 | x86_64 | [insiders-fast](./devops/README.md#red-hat-enterprise-linux-8---insiders-fast-amd64)
+Red Hat Enterprise Linux 9 | x86_64 | [insiders-fast](./devops/README.md#red-hat-enterprise-linux-9---insiders-fast-amd64)
+Rocky Linux 9 | x86_64 | [insiders-fast](./devops/README.md#rocky-linux-9---insiders-fast-amd64)
+SUSE Linux Enterprise Server 15 | x86_64 | [insiders-fast](./devops/README.md#suse-linux-enterprise-server-15---insiders-fast-amd64)
+Ubuntu 20.04 | x86_64 | [insiders-fast](./devops/README.md#ubuntu-2004---insiders-fast-amd64) / [prod](./devops/README.md#ubuntu-2004---prod-amd64)
+Ubuntu 22.04 | x86_64 | [insiders-fast](./devops/README.md#ubuntu-2204---insiders-fast-amd64) / [prod](./devops/README.md#ubuntu-2204---prod-amd64)
+
```
Comment (MariusNi) [inline README.md:37]: Move this to the end of the document.

Comment (MariusNi) [reply README.md:37]: Just before "Microsoft may collect performance and usage information which may be used ...."

File: devops/README.md [hunk 2]
```diff
@@ -1,17 +1,147 @@
...
+zypper ar -f https://packages.microsoft.com/yumrepos/microsoft-sles15-insiders-fast-prod/config.repo
+zypper install osconfig
+```
+
+## Ubuntu 20.04 - insiders-fast (AMD64)
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+wget https://packages.microsoft.com/config/ubuntu/20.04/insiders-fast.list -O /etc/apt/sources.list.d/packages-microsoft-com_insiders-fast.list
+apt update
+apt install osconfig
```
Comment (MariusNi) [inline devops/README.md:109]: Why apt and not apt-get?

File: devops/README.md [hunk 3]
```diff
@@ -1,17 +1,147 @@
+# Package Installation
```
Comment (MariusNi) [inline devops/README.md:1]: Can you also add package uninstallation? For each.

File: README.md [hunk 4]
```diff
@@ -2,12 +2,39 @@
...
+Debian 11 | x86_64 | [insiders-fast](./devops/README.md#debian-11---insiders-fast-amd64) / [prod](./devops/README.md#debian-11---prod-amd64)
+Debian 12 | x86_64 | [insiders-fast](./devops/README.md#debian-12---insiders-fast-amd64)
+Red Hat Enterprise Linux 7 | x86_64 | [insiders-fast](./devops/README.md#red-hat-enterprise-linux-7---insiders-fast-amd64)
+Red Hat Enterprise Linux 8 | x86_64 | [insiders-fast](./devops/README.md#red-hat-enterprise-linux-8---insiders-fast-amd64)
+Red Hat Enterprise Linux 9 | x86_64 | [insiders-fast](./devops/README.md#red-hat-enterprise-linux-9---insiders-fast-amd64)
+Rocky Linux 9 | x86_64 | [insiders-fast](./devops/README.md#rocky-linux-9---insiders-fast-amd64)
+SUSE Linux Enterprise Server 15 | x86_64 | [insiders-fast](./devops/README.md#suse-linux-enterprise-server-15---insiders-fast-amd64)
+Ubuntu 20.04 | x86_64 | [insiders-fast](./devops/README.md#ubuntu-2004---insiders-fast-amd64) / [prod](./devops/README.md#ubuntu-2004---prod-amd64)
+Ubuntu 22.04 | x86_64 | [insiders-fast](./devops/README.md#ubuntu-2204---insiders-fast-amd64) / [prod](./devops/README.md#ubuntu-2204---prod-amd64)
```
Comment (MariusNi) [inline README.md:36]: >-prod-amd64) [](http://example.com/codeflow?start=135&length=12)

At the end of this section, add:

"For instructions how to install the OSConfig poackage on each of these distros, see <link to the other README.md>"

File: devops/README.md (hunk 2 above)
Comment (AhmedBM) [reply devops/README.md:109]: No reason other than apt is available and usually more user-friendly

File: README.md (hunk 1 above)
Comment (AhmedBM) [reply README.md:37]: Sure, its a long document though, might get lost. I can add a TOC to make it more accessible?

File: README.md (hunk 4 above)
Comment (AhmedBM) [reply README.md:36]: the insiders-fast / prod links link directly to the instructions for the distro

Comment (AhmedBM) [reply README.md:36]: ohh i see what you mean now, ill describe that clicking on the insiders / prod links directly to the respective repo install/remove instructions

File: README.md [hunk 5]
```diff
@@ -2,12 +2,39 @@
...
 
-Azure Device OS Configuration (OSConfig) is a modular configuration stack for Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from Out Of Box Experience, OOBE).
+Azure Device OS Configuration (OSConfig) is a modular configuration stack for Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from Out Of Box Experience, OOBE).
 
 For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md) and [OSConfig Management Modules](docs/modules.md).
 
 For our code of conduct and contributing instructions see [CONTRIBUTING](CONTRIBUTING.md). For our approach to security see [SECURITY](SECURITY.md).
 
+## Pre-built Binary Packages
+We publish packages for all our supported platforms. See below for the list of supported platforms and the package repositories.
```
Comment (MariusNi) [inline README.md:12]: >We [](http://example.com/codeflow?start=0&length=2)

When moving, add one blank line between title and contents, same as the other sections do.

File: README.md [hunk 6]
```diff
@@ -2,12 +2,39 @@
...
+
+Channels | Description
+-----|-----  
+[![insiders-fast](https://github.com/Azure/azure-osconfig/actions/workflows/insiders-fast.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/insiders-fast.yml) | The latest and greatest features and bug fixes. This channel is updated frequently.
+[![prod](https://github.com/Azure/azure-osconfig/actions/workflows/prod.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/prod.yml) | The stable channel. This channel is updated less frequently and is recommended for production use.
+
+*If you see any missing platforms or architectures that would help, please contact osconfigcore@mcirosoft.com.
```
Comment (MariusNi) [inline README.md:19]: >mcirosoft.com [](http://example.com/codeflow?start=96&length=13)

Typo

File: devops/README.md [hunk 7]
```diff
@@ -1,17 +1,147 @@
...
+We use the [Microsoft repository](https://packages.microsoft.com/) to distribute our packages. Below are the instructions to install the `osconfig` package on various operating systems.
 
-This folder contains the artifacts and scripts used by our engineering systems (build, test, package).
+## Amazon Linux 9 - insiders-fast (AMD64)
+```bash
```
Comment (MariusNi) [inline devops/README.md:5]: Do a table instead of sections?

Table with 3 columns:

Distro | Installation | Uninstallation

File: README.md [hunk 8]
```diff
@@ -2,12 +2,14 @@
...
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](LICENSE.md)
 
-Azure Device OS Configuration (OSConfig) is a modular configuration stack for Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from Out Of Box Experience, OOBE).
+Azure Device OS Configuration (OSConfig) is a modular configuration stack for Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from Out Of Box Experience, OOBE).
 
 For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md) and [OSConfig Management Modules](docs/modules.md).
 
 For our code of conduct and contributing instructions see [CONTRIBUTING](CONTRIBUTING.md). For our approach to security see [SECURITY](SECURITY.md).
 
+For the list of our published binary packages and instructions how to install them see [here](./devops/README.md)
```
Comment (MariusNi) [inline README.md:11]: >here [](http://example.com/codeflow?start=88&length=4)

Why not "devops/README/md"? (instead of "here")

Comment (AhmedBM) [reply README.md:11]: sure, ill update

File: devops/README.md [hunk 9]
```diff
@@ -1,17 +1,211 @@
+# Package Installation
+We use the [Microsoft repository](https://packages.microsoft.com/) to distribute our packages. Below are the instructions to install the `osconfig` package on various operating systems.
```
Comment (MariusNi) [inline devops/README.md:2]: >We [](http://example.com/codeflow?start=0&length=2)

Add a blank line between the section titles and bodies. For consistency with the other MD documents in our project.

File: devops/README.md [hunk 10]
```diff
@@ -1,17 +1,211 @@
...
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/yumrepos/microsoft-amazonlinux2-insiders-fast-prod/config.repo
+yum install osconfig
+```
+
+## Amazon Linux 9 - Package Removal
+```bash
+yum remove osconfig
+```
```
Comment (MariusNi) [inline devops/README.md:38]: Why not in a table rather than many sections?

Comment (AhmedBM) [reply devops/README.md:38]: ill see if it looks better, standby

File: README.md (hunk 8 above)
Comment (MariusNi) [inline README.md:11]: >( [](http://example.com/codeflow?start=93&length=1)

Finish the sentence with a dot (add a missing '.' at the end)

File: devops/README.md (hunk 10 above)
Comment (MariusNi) [reply devops/README.md:38]: Take it back. Sections looks good.

**Review by MariusNi — DISMISSED** [review]
:shipit:

File: README.md [hunk 11]
```diff
@@ -2,12 +2,14 @@
...
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](LICENSE.md)
 
-Azure Device OS Configuration (OSConfig) is a modular configuration stack for�Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from Out Of Box Experience, OOBE).
+Azure Device OS Configuration (OSConfig) is a modular configuration stack for Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from Out Of Box Experience, OOBE).
 
 For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md) and [OSConfig Management Modules](docs/modules.md).
 
 For our code of conduct and contributing instructions see [CONTRIBUTING](CONTRIBUTING.md). For our approach to security see [SECURITY](SECURITY.md).
 
+For the list of our published binary packages and instructions how to install them see [/devops/README.md](./devops/README.md).
```
Comment (robertschaedler3) [inline README.md:11]: nit: remove the beginning `/` in the hyperlink text (we don't have a leading slash in other paths in this doc)

File: devops/README.md
Comment (robertschaedler3) [inline devops/README.md:1]: Thoughts on linking this doc to/from the MS Learn docs:

https://learn.microsoft.com/en-us/azure/osconfig/howto-install?tabs=package

File: devops/README.md [hunk 12]
```diff
@@ -1,17 +1,251 @@
...
 - `docker/<os>-<arch>` - Build containers for supported operating systems and architectures
   - `amd64`
   - `arm` (armv7)
   - `arm64` (aarch64)
-- `pipeline` - Azure Pipelines definitions
 - `e2e`
   - `cloudtest` - backup of 1ES managed CloudTest image definitions
   - `terraform` - terraform modules for provisioning Azure resources for E2E tests
-
-> See [Actions](https://github.com/Azure/azure-osconfig/actions) for more details and the status of our [workflows](https://github.com/Azure/azure-osconfig/tree/main/.github/workflows).
```
Comment (robertschaedler3) [inline devops/README.md:17]: Maybe add a badge to the top of this doc for the prod/insiders-fast workflows?

**Review by robertschaedler3 — APPROVED** [review]

**Review by MariusNi — DISMISSED** [review]
:shipit:

File: devops/README.md
Comment (AhmedBM) [reply devops/README.md:1]: Good idea, ill see if i can reference it somewhere although our content is more likely to be up-to-date

File: devops/README.md (hunk 12 above)
Comment (AhmedBM) [reply devops/README.md:17]: already did

**Review by MariusNi — APPROVED** [review]
:shipit:

**Pull request merged**